Xamarin.Forms.Material.iOS/MaterialMultiLineTextField.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using CoreGraphics;
+using MaterialComponents;
+using UIKit;
+using MMultilineTextField = MaterialComponents.MultilineTextField;
+using MTextInputControllerBase = MaterialComponents.TextInputControllerBase;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms.Platform.iOS.Material
+{
+	public class MaterialMultilineTextField : MMultilineTextField, IMaterialTextField
+	{
+		public SemanticColorScheme ColorScheme { get; set; }
+		public TypographyScheme TypographyScheme { get; set; }
+		public MTextInputControllerBase ActiveTextInputController { get; set; }
+		public ITextInput TextInput => this;
+		internal bool AutoSizeWithChanges { get; set; } = false;
+		CGSize _contentSize;
+
+
+		public MaterialMultilineTextField(IMaterialEntryRenderer element, IFontElement fontElement)
+		{
+			VisualElement.VerifyVisualFlagEnabled();
+			MaterialTextManager.Init(element, this, fontElement);
+		}
+
+		public override CGSize SizeThatFits(CGSize size)
+		{
+			bool expandTurnedBackOn = NumberOfLinesCheck();
+			var result = base.SizeThatFits(size);
+
+			if (nfloat.IsInfinity(result.Width))
+				result = SystemLayoutSizeFittingSize(result, (float)UILayoutPriority.FittingSizeLevel, (float)UILayoutPriority.DefaultHigh);
+
+			if (ExpandsOnOverflow)
+				_contentSize = result;
+			else
+				_contentSize = TextView.ContentSize;
+
+			if (!expandTurnedBackOn)
+				UpdateExpandsOnOverflow();
+
+			return result;
+		}
+
+		bool NumberOfLinesCheck()
```

Comment:
The method name does not match the content, instead of just checking it changes `ExpandsOnOverflow` variable.